ipad/ViewControllers/Login/LoginViewController.swift

```diff
@@ -80,6 +80,12 @@ class LoginViewController: BaseViewController {
     
     // MARK: Style
     private func style() {
+        if APIURL.baseURL == "https://api-dev-invasivesbc.apps.silver.devops.gov.bc.ca/api" ||
+            APIURL.baseURL == "https://api-test-invasivesbc.apps.silver.devops.gov.bc.ca/api" {
+            loginWithBCeIDButton.isHidden = false
```

Comment:
Again just pointing out BCEID stuff - might need to clean this up / make sure it still works for IDIR.